src/lib/mongoBackend/MongoCommonUpdate.cpp

```diff
@@ -81,6 +81,39 @@ using orion::CompoundValueNode;
 
 
 
+/* ****************************************************************************
+*
+* correlatorRoot -
+*
+* This functions returns the "correlator root", i.e. correlator without the "cbnotif" decorator
+*
+* For instance if full correlator is
+*
+*   f320136c-0192-11eb-a893-000c29df7908; cbnotif=32
+*
+* then the correlator root is:
+*
+*   f320136c-0192-11eb-a893-000c29df7908
+*
+* This is needed as the self-notification loop protection logic is based on the correlator root
```

Comment:
This modification (and other similar ones) make me to realize that self-notification could not be properly documented. It seems that only a small mention in data model document with regards to `lastCorrelator` field in DB mention it.

I should be described in some place of the documentation. Maybe in the one(s) related with custom notifications.

To be addressed within this PR.